pkg/stub/secrets.go

```diff
@@ -196,22 +196,10 @@ func (h *Handler) processSecretEvent(cfg *v1.Config, dockercfgSecret *corev1.Sec
 				return err
 			}
 		}
-		if removedState {
```

Comment:
for cross reference, this path should never have been entered, per https://github.com/openshift/cluster-samples-operator/blob/master/pkg/stub/handler.go#L561-L562